### PR TITLE
language_c7

### PR DESCRIPTION
**Pull request by thacuber2a03**
add a syntax highlighter for cel7 files, marked with .c7
it's fe's syntax highlighter, tailored to fit the things in cel7

oh and upgrade fe's syntax highlighter too

### CODE REVIEW

**Comment by thacuber2a03** [comment]
ah shit I messed up language_fe

**Comment by Guldoman** [comment]
Would you mind adding `language_c7` to `meta_languages`, and bumping that too?

**Comment by thacuber2a03** [comment]
k sure

**Comment by thacuber2a03** [comment]
I thought of bumping it a minor, but idk...

don't take this out of context

File: manifest.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     {
       "id": "meta_languages",
       "type": "meta",
-      "version": "0.1.1",
+      "version": " 0.1.2",
```
Comment (Guldoman) [inline manifest.json:64]:
```suggestion
      "version": "0.1.2",
```

Comment (thacuber2a03) [reply manifest.json:64]: HOW DID THAT GET THERE HAHAHAHAHAH

File: plugins/language_c7.lua [hunk 2]
```diff
@@ -0,0 +1,80 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "cel7",
+  files = "%.c7$",
+  comment = ";",
+  patterns = {
+    { pattern = ";.-\n",              type = "comment"  },
```
Comment (Guldoman) [inline plugins/language_c7.lua:9]:
```suggestion
    { pattern = ";.*",                type = "comment"  },
```
Could you do the same for `language_fe`?

**Pull request merged**